Index: Makefile
===================================================================
--- Makefile	(revision 525)
+++ Makefile	(working copy)
@@ -9,10 +9,10 @@
 WINDRES=windres
 CFLAGS=-Wall -D_GNU_SOURCE -O2
 CXXFLAGS=$(CFLAGS)
-LDLIBS=
+LDLIBS=-lTomiTCP
 LDFLAGS=
-REVISIONS=-DREVISION=\"$(shell svn info | perl -ne 'if(/Last Changed Rev: (\d+)/){print $$1;}')\" \
-	-DTOMITCP_REV=\"$(shell svn info TomiTCP | perl -ne 'if(/Last Changed Rev: (\d+)/){print $$1;}')\"
+REVISIONS=-DREVISION=\"$(shell svn info ${SVN_DIR}/xchat-bttrw/head| perl -ne 'if(/Last Changed Rev: (\d+)/){print $$1;}')\" \
+	-DTOMITCP_REV=\"$(shell svn info ${SVN_DIR}/TomiTCP/head | perl -ne 'if(/Last Changed Rev: (\d+)/){print $$1;}')\"
 RSRC=
 
 LINK.o=$(CXX) $(LDFLAGS) $(TARGET_ARCH)
@@ -66,7 +66,7 @@
                   charset.o list.o ison.o userinfo.o superadmins.o
 	$(AR) rsv $@ $?
 
-gate: gate.o setproctitle.o libxchat-bttrw.a TomiTCP/libTomiTCP.a $(RSRC)
+gate: gate.o setproctitle.o libxchat-bttrw.a $(RSRC)
 	$(LINK.o) $^ $(LOADLIBES) $(LDLIBS) -o $@
 ifeq ($(TARGET),i386-mingw32msvc)
 	strip $@
@@ -79,9 +79,6 @@
 	$(RM) libxchat-bttrw.a gate *.o
 #	$(MAKE) -C TomiTCP clean
 
-TomiTCP/libTomiTCP.a: dummy
-	$(MAKE) -C TomiTCP DEBUG=$(DEBUG)
-
 %.o: %.rc
 	$(WINDRES) -o $@ $^
 
Index: gate.cc
===================================================================
--- gate.cc	(revision 525)
+++ gate.cc	(working copy)
@@ -317,7 +317,7 @@
 			fprintf(*c, ":%s 375 %s :- %s Message of the Day -\n", me,
 				nick.c_str(), me);
 
-			ifstream copyright("COPYRIGHT");
+			ifstream copyright("/usr/share/xchat-bttrw/COPYRIGHT");
 			if (copyright) {
 			    string motd;
 			    while (getline(copyright, motd))
